DEVGUIDE.md

```diff
@@ -44,12 +48,13 @@ Run below to genreate API documentation. This takes the `APIReference.hbs` and t
 npm run build:docs
 ```
 
-## Releasing
+## Publishing to npm registry
 
 We are using [npm-publish](https://github.com/JS-DevTools/npm-publish) to handle publishing.
 So to publish a release to NPM, we need to 
 - Change the version in `package.json` as needed.
 - Run `npm install` (this will update `src/version.ts` file).
```

Comment:
should this be `npm ci`

Reply:
@ivansenic Good point. `npm ci` requires `package-lock.json` file which we decided not to check in. Please see [here](https://github.com/stargate/stargate-mongoose/pull/51#issuecomment-1470168606).
Probably I can change this to `rm -rf node_modules && npm install`

@vkarpov15 Does the above look good for a release process without `npm ci`, which requires `package-lock.json`?

Reply:
I would advise keeping `npm install` over `npm ci`. @kathirsvn is right that `npm ci` requires `package-lock.json`, and `package-lock.json` is not useful for packages that are published to npm. Because if stargate-mongoose publishes a `package-lock.json` file, that `package-lock.json` file is ignored when a client runs `npm install`. The client's `package-lock.json`, if any, is the only one that npm uses.